src/component/Auth/Auth.jsx

```diff
@@ -0,0 +1,75 @@
+import { useFormik, form } from "formik";
+import { addUser } from "../../util/firebase";
+import { useNavigate } from "react-router-dom";
+
+import { TextField, Button } from "@mui/material";
+import { useState } from "react";
+
```

Comment:
When a user clicks on the google signin it gives the a redirected page with google login option and when the token comes back it stores the token in the local storage and redirect again to the /Home